Blob Nerf

KathrinBailey opened the pull request:
## About The Pull Request
**Nerfs:**
• Blob Core health 800 => 600 (originally 400).
• Blob Factory health 400 => 300 (originally 200); max spores 5 => 3.
• Blob Node health 400 => 150 (originally 200).
• Blob Resource health 120 => 80 (original 60).
• Normal Blob tile health 60 => 40 (originally 30).
• Blob resource generation reverted from 2x => 1x.
• Attack cost is now 2 resources (this is a buff because it was actually 3 resources with Jake’s nerf, which was a bug).
• Shield Blob upgrade cost 5 => 10 (originally 15).
• Blob spore cloud size reverted from 2 => 1.
• Blob node distance from other nodes reverted from 4 => 5.
• Blob resource distance from other resource blobs reverted from 3 => 4.
• Reverts blob points generation from 2 => 1.
• Reverts node blob free upgrades to surrounding blobs to strong blobs from 5% => 0.1% as it was just spitting out upgrades constantly.

**Buffs:**
• Spore lower damage 2 => 5; spore upper damage 4 => 10.
• Resource Blob upgrade cost 30 => 25 (originally 40).
**Quality of Life:**
• Blobbernaut tooltip tells the player that blobbernauts are player-controlled.
• Factory tooltip now displays the correct price for factory blobs.
• Resource tooltip now displays the correct price for resource blobs.

## Why It's Good For The Game
This is a partial revert of #3228 and nerf to the changes of #1355. The best parts of #3228 were kept whilst #1355's health buffs that make the blob so frustrating to fight have been reeled back. Think of it as a revert of #3228 that brings back some of it, so it's a minor buff to what we had beforehand and a moderate nerf to what we have now. It was a good change, and definitely better change than outright doubling blob health, but wounds for some reason being given for 20 force attacks (the fuck?) and 

Having doubled health made the blobs incredibly annoying to fight. Emitters, which used to carve the blob up, became kinda laughable.

## What do I do if I'm a player?
If you're a player looking at this PR and there's a blob, do one of these things:
• Get roller beds and medical equipment, drag people out and keep them alive.
• Break into security checkpoints in departments and unwrench the weapon rechargers, drag them towards the blob so those with lasers can keep firing.
• Scientists should mass-print experimental welders and throw them for the masses to collect.
• Engineers and anyone with EVA access should set emitters up facing the blob core in space, away from the blob and they should protect said emitters.
• If engineering is incompetent or scarce, break in and steal emitters from the supermatter engine **but not the damn singularity engine** and set them up to carve up the blob. Deconstructing the reinforced wall into secure engineering will help with this.
• Miner? Get a PKA and vent the area completely. Use points to supply the crew with more PKAs. The insane damage and infinite ammo is invaluable.
• Drag water and fire extinguishers to the blob. If they decide to change the strain into one vulnerable to water (blazing oil), **punish**.

## Changelog
:cl: Floof Ball#0798 Kathrin Morrison
balance: Blobs are now weaker, with 1.5x resource generation, blob spores are stronger, resource nodes are cheaper and the health of blobs has been lowered across the board to make them less frustrating to fight.
fix: Blob tooltips were displaying incorrect prices for blob tiles and missing some critical information.
/:cl:

KathrinBailey committed:
Blob nerfs

H0gberg commented:
Original buff was a good step in the right direction but too stronk this is a good PR. Smart floof

jjpark-kb commented:
good PR.

KathrinBailey committed:
It's still a bit too buffed.

Owai-Seek commented:
In regards to blob antagonist itself, and not the balance of the PR, I just want to say something.

The biggest problem I have with blobs from a player standpoint (for me) is that it interrupts any and all roleplay that's going on, and you -have- to fight the threat, even if you're not well equipped to deal with it. It's not fun when you get pulled out of whatever shit you're doing to fight a mass of flesh. It also spawns incredibly often (in comparison, to say, xenos.)

I don't know what to say balance-wise for it, because honestly, I'm just sick of dealing with them in general. I also feel some of these tips don't hold up with the constant changes bobmed brings.

- Bobmed now give you injuries just for being in pressurized environments (spaced areas) without protection. This means if you have no hardsuit, EVA gear, or firesuit, you can't participate in the fight , even from a distance, without getting wounds.
- Getting in melee range means you're going to have the blob itself, or blobbernauts attacking you, which is almost a guaranteed wound or two. Get a foot or leg wound and you might not be able to escape.
- Grabbing people and keeping them alive is extremely dangerous and hard to do, especially when blobbernauts are beating the shit out of them to make sure they're dead and have an entire stack of wounds on them.
- Breaking into engineering kinda invalidates the rules about knowing a single department. A character with no engineering knowledge would have no idea what emitters are for, or how to set them up.

Before this buff, blobs were just barely getting destroyed, requiring multiple mechs and people firing into it to put a dent into things, and even in those scenarios, it was because blobbernauts were retreating too early.

I'm all for giving this a test to see how it goes, but honestly, I'm just so sick of blobs completely taking over any and every story going on in a round because if it isn't dealt with, the round is over, and even if it -is- dealt with, it often results in a shuttle call because of massive damage to the station.

jjpark-kb commented:
> In regards to blob antagonist itself, and not the balance of the PR, I just want to say something.
> 
> The biggest problem I have with blobs from a player standpoint (for me) is that it interrupts any and all roleplay that's going on, and you -have- to fight the threat, even if you're not well equipped to deal with it. It's not fun when you get pulled out of whatever shit you're doing to fight a mass of flesh. It also spawns incredibly often (in comparison, to say, xenos.)
> 

I mean, thats just gonna happen. Events are going to happen whether you like it or not. Depending on the severity, will require people to jump into action to help resolve the situation. Those events are going to happen whether one person doesn't want it. That is part of the story. You don't ever have to fight the threat if its a station-wide threat, but you should be there to assist. Assisting such as bringing in weapon chargers, bringing dead bodies to medical, bringing medicine to keep people in the fight, etc. You don't have to get involved in combat if you don't want to. You can contribute to the storyline by doing those things, which help A LOT.

Xenos are also way harder to defeat if they go unnoticed. People normally believe that the infestation alert is just spiders or a false alarm. No one really expects the xenos unless one of them fucks up. Xenos are way stronger, so it appears less.

> I don't know what to say balance-wise for it, because honestly, I'm just sick of dealing with them in general. I also feel some of these tips don't hold up with the constant changes bobmed brings.
> 
> * Bobmed now give you injuries just for being in pressurized environments (spaced areas) without protection. This means if you have no hardsuit, EVA gear, or firesuit, you can't participate in the fight , even from a distance, without getting wounds.
> * Getting in melee range means you're going to have the blob itself, or blobbernauts attacking you, which is almost a guaranteed wound or two. Get a foot or leg wound and you might not be able to escape.
> * Grabbing people and keeping them alive is extremely dangerous and hard to do, especially when blobbernauts are beating the shit out of them to make sure they're dead and have an entire stack of wounds on them.
> * Breaking into engineering kinda invalidates the rules about knowing a single department. A character with no engineering knowledge would have no idea what emitters are for, or how to set them up.
> 

yea, a lot of this isn't a blob issue in my opinion. This is stuff interacting with another PR. I think bob should make blobs and blobbernauts not give wounds when attacking since they already hurt, so yea, bring that up with Bob. I dont think its very appropriate here.

> Before this buff, blobs were just barely getting destroyed, requiring multiple mechs and people firing into it to put a dent into things, and even in those scenarios, it was because blobbernauts were retreating too early.
> 

This PR isnt an overall buff, its an overall nerf. My PR to buff blob is because blob was laughably weak. It wasnt just barely getting destroyed, it was getting utterly destroyed. Almost nothing the blob could do would help it once the crew started to swarm it. 

Blobbernauts retreating too early, yep. This is a player issue, where someone isn't good at a role. The same can be said for the crew that don't know how to deal with current blobs and are inept/incompetent in regards to dealing with them.

> I'm all for giving this a test to see how it goes, but honestly, I'm just so sick of blobs completely taking over any and every story going on in a round because if it isn't dealt with, the round is over, and even if it -is- dealt with, it often results in a shuttle call because of massive damage to the station.

Well, Im glad you are fine for testing this PR, since this is a nerf, not a buff. It is up to the crew if they want to call the shuttle. If the engineers don't feel like repairing, thats their fault, not the blobs.

Owai-Seek commented:
When I refer to buff, I refer to the original PR that buffed the blob, not this one.

YakumoChen commented:
>Well, Im glad you are fine for testing this PR, since this is a nerf, not a buff. It is up to the crew if they want to call the shuttle

:blobgrin:

In all seriousness though good PR. I played blob once and it really was a cutbstomp, but pre buff was just not playable with a decently staffed crew. Resource management being scarce was the biggest pain point towards play and this is a good step, making blob just hilariously hanky was not.

MLGTASTICa commented:
Way too much nerfing , reverting the additional resource regen and all the health buffs is a MASSIVE nerf. What deserves nerfing was health, as long as the additional resource regen was kept. With all these nerfs the blob will have the old issue of never having enough resources with the added stress of weakened blob types.

KathrinBailey committed:
Slightly buffs resource generation.

Health was reduced and point generation went from 2 with Jake's buffs to 1, now it's at 1.5 which should be a good place.
